Update CONTRIBUTING after cloud-scripts integration

eskombro opened the pull request.

curquiza reviewed (commented):
Shouldn't we remove the part about adding a tag in the test section (Test before Releasing)?
I thought the tag is not useful anymore during the test phase, and we decided to put a tag only for the release so that is cleaner (but not mandatory).

eskombro commented:
> Shouldn't we remove the part about adding a tag in the test section (Test before Releasing)?
> I thought the tag is not useful anymore during the test phase, and we decided to put a tag only for the release so that is cleaner (but not mandatory).

Done. Indeed a few instructions changed in this process. I modified the whole flow, hope everything is clear! Also, I added the "set your environment" instructions :)

curquiza reviewed (approved):
Like this! Seems clear to me! :)
